web-ng/src/app/components/map/map.component.ts

```diff
@@ -294,6 +326,34 @@ export class MapComponent implements AfterViewInit, OnDestroy {
     });
   }
 
+  private addPolygonToMap(project: Project, feature: PolygonFeature) {
+    const color = project.layers.get(feature.layerId)?.color;
+    const vertices: google.maps.LatLng[] = [];
+    feature.polygonVertices.map(vertex => {
+      vertices.push(new google.maps.LatLng(vertex.latitude, vertex.longitude));
+    });
+    // Construct the polygon.
+    const polygon = Polygon(feature.id , new google.maps.Polygon({
+      paths: vertices,
+      clickable: true,
+      strokeColor: color,
+      strokeOpacity: 1,
+      strokeWeight: 4,
+      fillOpacity: 0,
+    }));
+    polygon.addListener('click', () => {
+      polygon.setOptions({ strokeWeight: 6 });
```

Comment:
We should extract this to an constant. 
Do a pull from upstream first. I merged a PR which introduced these two consts in this component:
```
const normalPolygonStrokeWeight = 3;
const enlargedPolygonStrokeWeight = 6;
```
You can reuse them.

Reply:
added 
`
const normalPolygonStrokeWeight = 3;
const enlargedPolygonStrokeWeight = 6;
`